modules/virt-configuring-highburst-profile.adoc

```diff
@@ -0,0 +1,19 @@
+
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-vm-control-plane-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-highburst-profile_{context}"]
+= Configuring a highBurst profile
+
+Use the `highBurst` profile to create and maintain a large number of virtual machines (VMs) in one cluster.
+
+.Procedure
+
+. Apply the following patch to enable the `highBurst` `tuningPolicy` profile.
```

Comment:
It looks odd to have so many terms in backticks. I recommend using generic language for the profile description:

"Enable the `highBurst` tuning policy profile by running the following command:"